src/clj/rems/application/commands.clj

```diff
@@ -392,9 +398,10 @@
   [cmd application injections]
   (or (already-member-error application (:actor cmd))
       (invitation-token-error application (:token cmd))
-      (ok {:event/type :application.event/member-joined
-           :application/id (:application-id cmd)
-           :invitation/token (:token cmd)})))
+      (ok-with-data {:application-id (:application-id cmd)}
```

Comment:
why do you need to return the app id? it was specified in the cmd so the sender knows it.

Reply:
That simplifies the code in https://github.com/CSCfi/rems/pull/1512/files#diff-5045c91462d19bb329c2d32136b4f565R137-R140 by using common code instead of the old custom solution https://github.com/CSCfi/rems/pull/1512/files#diff-6abc0721f32efbf6b854ba6af49ae23fL134-L143